scripts/firedrake-install

```diff
@@ -301,6 +316,17 @@ def git_update(name):
         else:
             check_call(["git", "checkout", travis_commit])
     else:
+        # Ensure remotes get updated if and when we move repositories.
+        if url:
+            plain_url = split_requirements_url(url)[1]
+            current_url = check_output(["git", "remote", "-v"]).split()[1]
+            protocol = "https" if current_url.startswith("https") else "ssh"
+            new_url = git_url(plain_url, protocol)
+            # Ensure we only change from bitbucket to github and not the reverse.
+            if new_url != current_url and "bitbucket.org" in current_url \
+               and "github.com/firedrakeproject" in plain_url:
```

Comment:
So this works because plain_url is always `/` separated (because it comes from the requirements file).